packages/dso-toolkit/src/components/application-heading/application-heading.mixins.scss

```diff
@@ -23,7 +28,7 @@
       flex-basis: auto;
       font-size: 1rem;
       font-weight: normal;
-      padding-top: #{application-heading-variables.$step-padding * 0.5};
+      padding-block-start: #{application-heading-variables.$step-padding * 0.5};
```

Comment:
Ik zie dat die step-padding 12 pixels is, maar verder nergens wordt gebruikt dan hier in de berekening. is het dan niet gemakkelijker om die step-padding 6 pixels te maken en hier die berekening te vervangen door alleen die variabele?

Reply:
Klinkt logisch, maar dat hoort niet in deze PR. Het enige dat ik hier gedaan heb, is omzetten naar een logical property (bijvangst) :)

Reply:
Tsja... Maar waar ligt de grens ;-) Had je misschien beter helemaal niets kunnen doen.

En volgensn mij hebben we het daar laatst ook over gehad: We gaan alleen logical properties introduceren als we er mee bezig zijn. Deze bijvangst introduceert nu ruis.

Laten we het omkatten van physical naar logical in een losse issue doen.

Reply:
We hebben ooit afgesproken dat als we een bestand aanraken, we daar ook meteen van physical naar logical gaan.

Reply:
Ik meen mij te herinneren dat die afspraak niet over het hele bestand ging maar over de daadwerkelijke code die wordt aangeraakt.

Maar dit zal niet onze eerste miscommunicatie zijn (ahem #2357).

Zijn we het erover eens dat deze changes niets toevoegen aan het PR, en in dit style-only PR zelfs ruis toevoegen?